projects/jones-dao/index.js

```diff
@@ -1,89 +1,100 @@
+      abi: "uint256:workingBalance",
+      calls: addresses.metaVaultsAddresses,
+    }),
+    api.multiCall({
+      abi: "address:asset",
+      calls: addresses.optionVaultAddresses,
+    }),
+    api.multiCall({
+      abi: "uint256:totalAssets",
+      calls: addresses.optionVaultAddresses,
+    }),
+  ]);
 
-  const toa = []
+  api.addTokens(metavaultTokens, metavaultBalances);
+  api.addTokens(optionVaultTokens, optionVaultBalances);
 
-  api.addTokens(tokens,bals)
-  api.addTokens(vAssets,vBals)
-  api.addToken(addresses.trackers.uvert.token, uvrtBalance)
-  Object.values(addresses.trackers).map(tracker => toa.push([tracker.token, tracker.holder]))
-  toa.push([addresses.glp, addresses.strategy,])
+  const tokensAndOwners = [
+    [addresses.tokens.uvrt, addresses.glp.stableRewardTracker],
+    [addresses.tokens.uvrt, addresses.glp.router],
+    [addresses.tokens.glp, addresses.glp.leverageStrategy],
+  ];
 
-  return sumTokens2({ api, tokensAndOwners: toa, });
+  return sumTokens2({ api, tokensAndOwners });
 }
 
-async function tvl_ethereum(timestamp, block, chainBlocks, { api }) {
-  const balances = {}
+async function tvl_ethereum(_timestamp, _block, _chainBlocks, { api }) {
+  const balances = {};
 
-  const leftoverStrategy = await sdk.api.erc20.balanceOf({
-    target: AURA,
-    owner: jAuraStrategy,
-    chain: 'ethereum',
-    block: chainBlocks['ethereum']
-  }).then(result => result.output)
-  sdk.util.sumSingleBalance(balances, AURA, leftoverStrategy)
+  const leftoverStrategy = await sdk.api.erc20
+    .balanceOf({
+      target: addresses.tokens.aura,
+      owner: addresses.aura.strategy,
+    })
+    .then((result) => result.output);
+  sdk.util.sumSingleBalance(balances, addresses.tokens.aura, leftoverStrategy);
 
-  const lockedBalance = await sdk.api.abi.call({
+  const lockedBalance = await sdk.api.abi
+    .call({
       abi: lockerABI.at(0),
-      target: AURALocker,
-      params: jAuraStrategy,
-      chain: 'ethereum',
-      block: chainBlocks['ethereum']
-  }).then(result => result.output[0])
-  sdk.util.sumSingleBalance(balances, AURA, lockedBalance)
+      target: addresses.aura.locker,
+      params: addresses.aura.strategy,
+    })
+    .then((result) => result.output[0]);
+  sdk.util.sumSingleBalance(balances, addresses.tokens.aura, lockedBalance);
 
   return balances;
 }
 
 module.exports = {
   arbitrum: {
     tvl,
-    pool2: pool2s(addresses.lpStaking, addresses.lps, "arbitrum", addr => {
+    pool2: pool2s(addresses.lpStaking, addresses.lps, "arbitrum", (addr) => {
       addr = addr.toLowerCase();
-      if (jTokenToToken[addr] !== undefined) {
-        return jTokenToToken[addr];
-      }
-      return `arbitrum:${addr}`;
+      return `arbitrum:${jAssetToAsset[addr] ?? addr}`;
     }),
-    staking: stakings(addresses.jonesStaking, addresses.jones)
+    staking: stakings(
+      addresses.stakingContracts,
+      [
+        addresses.tokens.jones,
+        addresses.tokens.jglp,
```

Comment:
hi @0xIvan 
thanks for the PR, it is mostly good, but isnt the back backing of jglp, jusdc and wjaura already counted as part of tvl?

Reply:
if so, counting it under staking would be doublecounting